modules/ROOT/pages/fleet-management/overview.adoc

```diff
@@ -0,0 +1,30 @@
+= Overview
+
+Get an enterprise-wide view of all your Neo4j instances, both Aura instances and/or self-managed.
+
+For users operating in hybrid environments, it’s common to have a mix of Aura instances and self-managed Neo4j deployments. 
+Integrating self-managed deployment into the Aura console allows for a more centralized management.
+
+== Centralized management
+Registering self-managed Neo4j deployments in the Aura Console allows you to monitor all your Neo4j databases in one place.
+
+
+== Security 
+
+Fleet management is designed to be secure, no inbound connections to your self-managed deployment are allowed.
+Further, there is no capability for performing any write operations or other mutations remotely.
+
+No data is read from your databases.
+See xref:fleet-management/data.adoc[Data transparency] for a complete list of the data sent to Aura from your self-managed deployment.
+
+== Features
+
+Fleet management offers various features to manage and monitor your self-managed Neo4j deployments:
+
+* Metrics
+* Topology views (graph and tabular format)
+* Server configuration viewer
+* Database metadata
+* License status viewer
+
+In addition to these features, the user tools Query, Explore, and Dashboards (available in the Aura console) are also available for self-managed deployments.
```

Comment:
@AlexicaWright Your previous comment about write actions disappeared, but the user tools use their own bolt connections. All we provide to them is the URL for connecting. So they can perform writes as well, but the customer must configure their firewall accordingly, and provide credentials when connecting.

Reply:
I see, so it's only from an EFM perspective that no writes are allowed?

Reply:
Exactly. The EFM connection cannot write anything. Using the user tools involves establishing a separate bolt connection, which does not come with that limitation.